sdk/textanalytics/azure-ai-textanalytics/tests/mock_test_responses/action_error_with_targets.json

```diff
@@ -8,12 +8,12 @@
     {
       "code": "InvalidRequest",
       "message": "Some error2",
-      "target": "#/tasks/entityRecognitionPiiTasks/0"
+      "target": "#/tasks/piiEntityRecognitionTasks/0"
     },
     {
       "code": "InvalidRequest",
       "message": "Some error5",
-      "target": "#/tasks/entityRecognitionPiiTasks/1"
+      "target": "#/tasks/piiEntityRecognitionTasks/1"
```

Comment:
@kristapratico  Does this `piiEntityRecognitionTasks` change intended in purpose?  I can't find the keyword generated in the latest implementation code.

Reply:
yes, manually updated to match what the service returns